addons/website/models/website.py

```diff
@@ -468,7 +468,15 @@ def configure_page(page_code, snippet_list, pages_views, cta_data):
             page_view_id.save(value=''.join(rendered_snippets), xpath="(//div[hasclass('oe_structure')])[last()]")
 
         def set_images(images):
+            names = self.env['ir.model.data'].search([
+                ('name', '=ilike', f'configurator\_{website.id}\_%'),
```

Comment:
@SonCrits Oops, my mistake <https://runbot.odoo.com/runbot/batch/1111693/build/48045133>: `\` itself is supposed to be escaped within the f-string.
=> `f'configurator\\_{website.id}\\_%'`
Sorry about this additional iteration.

Reply:
thank you bro @bso-odoo